xbmc/utils/CryptThreading.cpp

```diff
@@ -46,7 +56,11 @@ CryptThreadingInitializer::CryptThreadingInitializer()
   // OpenSSL < 1.1 needs integration code to support multi-threading
   // This is absolutely required for libcurl if it uses the OpenSSL backend
   m_locks.resize(CRYPTO_num_locks());
+#if KODI_OPENSSL_USE_THREADID
```

Comment:
```suggestion
# if KODI_OPENSSL_USE_THREADID
```

Reply:
@pkerling Why add all that white space?

Reply:
Please not! 

As the directives don't care about any level!

Reply:
There is nothing in the code guidelines but most of the kodi code does not use indentation.